Makefile

```diff
@@ -34,25 +34,20 @@ OUT_DIR=internal/api
 proto:
 	protoc --proto_path=$(PROTO_DIR) --go_out=$(OUT_DIR) --go-grpc_out=$(OUT_DIR) --golang-deepcopy_out=:$(OUT_DIR) $(PROTO_DIR)/**/*.proto
 
-.PHONY: gen-istio-manifests
-gen-istio-manifests:
-	bash test/scripts/generate_istio_manifests.sh $(ISTIO_VERSION)
-
 .PHONY: kind-clusters
 kind-clusters:
 	bash test/scripts/kind_provisioner.sh $(ISTIO_VERSION)
 
-.PHONY: e2e-test
-e2e-test:
-	go test -tags=integ -run TestTraffic ./test/e2e \
-		--istio.test.hub=docker.io/istio\
-		--istio.test.tag=$(ISTIO_VERSION)\
-		--istio.test.kube.config=$(shell pwd)/test/east.kubeconfig,$(shell pwd)/test/west.kubeconfig\
-		--istio.test.kube.networkTopology=0:east-network,1:west-network\
-		--istio.test.onlyWorkloads=standard
-
 .PHONY: e2e
```

Comment:
> backward compat (thinking of CI jobs)

Currently, we run `make e2e` in the prow job.

I will leave it as is, because if I changed it to `e2e-test` I would have to update our prow jobs, and sometimes this is very time consuming.